pageserve/app.py

```diff
@@ -52,11 +52,34 @@ def index():
         'index.html',
         posts=posts,
         auth=is_auth,
+        user=user,
         app_config=app.config
     )
 
 
-@app.route('/v1/events')
+@app.route('/v1/add_post', methods=['POST'])
+def add_post():
+    """Add post by calling posts service."""
+    url = app.config['POSTS_ENDPOINT'] + 'add'
+    r = requests.post(url, data=request.form)
+    if r.status_code == 201:
+        return "Post successfully added"
+    else:
+        return r.content
+
+
+@app.route('/v1/add_event', methods=['POST'])
+def add_event():
```

Comment:
Does the ordering of this file make sense?

Right now, it looks like:

* config_endpoints
* GET /v1
* POST /v1/add_post
* POST /v1/add_event
* GET /v1/events
* get/parse posts
* get/parse events
* get_user_info

Reply:
A few of our other files also have quite poor ordering. I think it might make sense to standardize something like
- Routes alphabetized by endpoint then method
- Helper functions alphabetized
- Configuration functions (config endpoints, connect to db) alphabetized
- if __name__ == "__main__"

Reply:
Alphabetizing might not make the most logical ordering of functions, depending on naming. For now I'll re-order the endpoints and move the config functions.

Wondering if it's worth another PR to re-order files?

Reply:
This sort of refactor is often good as a fast-followup in a separate PR when it moves existing functions (so that the reviewer can easily distinguish between changed/added code and simple cleanups).

The amount of change in this PR isn't a big deal, but if you have (for example) a Java class with 20 methods and you reorganize them while adding 2 more, it can get tricky to review.